services/director/src/simcore_service_director/registry_proxy.py

```diff
@@ -62,13 +63,13 @@ async def _basic_auth_registry_request(
     url = URL(
         f"{'https' if app_settings.DIRECTOR_REGISTRY.REGISTRY_SSL else 'http'}://{app_settings.DIRECTOR_REGISTRY.REGISTRY_URL}{path}"
     )
-    logger.debug("Requesting registry using %s", url)
+    _logger.debug("Requesting registry using %s", url)
     # try the registry with basic authentication first, spare 1 call
     resp_data: dict = {}
     resp_headers: Mapping = {}
     auth = (
-        BasicAuth(
-            login=app_settings.DIRECTOR_REGISTRY.REGISTRY_USER,
+        httpx.BasicAuth(
+            username=app_settings.DIRECTOR_REGISTRY.REGISTRY_USER,
             password=app_settings.DIRECTOR_REGISTRY.REGISTRY_PW.get_secret_value(),
```

Comment:
MINOR: perhaps you can create a property within e.g. app_settings.DIRECTOR_REGISTRY.is_authenticated

Reply:
I reworked the registry settings. but I find that is_authenticated is too much. there is already REGISTRY_AUTH